ONSdigital/response-operations-ui
# RAS-939 Survey Status Change

# What and why?
Adds a new case group transition from `complete` to `not started` to allow business users a new status change in rOps
# How to test?
- deploy this and https://github.com/ONSdigital/rm-case-service/pull/289 to your environment
- run the acceptance tests to populate your env
- log into frontstage and complete the survey
- go to http://localhost:8085/reporting-units/49900000001 and navigate to change the status from complete to not started
- check that the survey appears in the todo page in frontstage
- check that http://localhost:8085/reporting-units/49900000001 and http://localhost:8085/case/49900000001/response-status?survey=QBS&period=1912 look the same and act the same as they do on main including with and without permissions
# Jira
https://jira.ons.gov.uk/browse/RAS-939
